---
layout: default
---

		<!-- Banner -->		
			<section id="banner">
				
				<!--
					".inner" is set up as an inline-block so it automatically expands
					in both directions to fit whatever's inside it. This means it won't
					automatically wrap lines, so be sure to use line breaks where
					appropriate (<br />).
				-->
				<div class="inner">
					
					<header>
						<h2>ENVGEN</h2>
					</header>
					<p>Welcome to <strong>EnvGen</strong>, the group<br />
					of Environmental Genomics<br />
                    at <a href="http://kth.se">KTH</a>, located at <a href="http://scilifelab.se">Scilifelab</a><br />
                    in Stockholm, Sweden.</p>
					<footer>
						<ul class="buttons vertical">
							<li><a href="#main" class="button fit scrolly">Tell Me More</a></li>
						</ul>
					</footer>
				
				</div>
				
			</section>
		
		<!-- Main -->
			<article id="main">

				<header class="special container">
					<span class="icon fa-bar-chart-o"></span>
                    <h2> We study <strong>microbial communities</strong> using shotgun metagenomics and amplicon <strong>sequencing</strong> and develop methodology for enhancing such studies.
                    </h2>
                    <p>
                    The activities of complex microbial communities shape our planet and affect the health of the humans. In our group we employ metagenomics and taxonomic marker gene sequencing to explore patterns of microbial diversity and reconstruct genomes and metabolic pathways of microbes in environments ranging from the Baltic Sea to the rumen of Moose. We also develop molecular and bioinformatics tools for enhancing the information output from such studies. An introduction to metagenomics and amplicon sequencing can be found <a href="/metagenomics.html#main">here</a>.
                    </p>
				</header>
					
				<!-- One -->
					<section class="wrapper style3 container special", id="who">
					
						<header class="major">
							<h2>Who we are</h2>
						</header>
						
						<div class="row">
							<div class="6u">
							
								<section>
									<a href="#" class="image feature"><img src="images/AndersA.jpg" alt="" /></a>
									<header>
										<h3>Anders Andersson (PI)</h3>
									</header>
                                    <p>
                                        Anders received his PhD in 2005 at KTH Royal Institute of Technology after graduate studies on functional genomics.  Thereafter he conducted postdoctoral studies at the Karolinska Institute, University of California Berkeley, and Uppsala University, working on metagenomics approaches to study microbial communities of different environments (the human gut microbiota, biofilms of acid mine drainage, and bacterioplankton of lakes and the Baltic Sea). In 2010 he returned to KTH, starting his tenure track and building up a group at Science for Life Laboratory in Stockholm. In 2014 he was promoted to associate professor.
                                    </p>
								</section>

							</div>
							<div class="6u">
							
								<section>
									<a href="#" class="image feature"><img src="images/luisa.jpg" alt="" /></a>
									<header>
										<h3>Luisa W. Hugerth</h3>
									</header>
									<p>I am a PhD student working on microbial communities in the Baltic Sea. We use shotgun sequencing to reconstruct genomes from uncultured organisms and explore their metabolism. As a complement to that, we use amplicon sequencing to investigate population dynamics over time. </p>
                                    <p>In collaboration with other groups, I'm exploring these same approaches in other environments, such as soils and the human gut.</p>
								</section>

							</div>
						</div>
						<div class="row">
							<div class="6u">
							
								<section>
									<a href="#" class="image feature"><img src="images/ino.jpg" alt="" /></a>
									<header>
										<h3>Ino de Bruijn</h3>
									</header>
									<p>The assembly and bash pirate.</p>
								</section>

							</div>
							<div class="6u">
							
								<section>
									<a href="#" class="image feature"><img src="images/johannes.jpg" alt="" /></a>
									<header>
										<h3>Johannes Alneberg</h3>
									</header>
                                    <p> With a background in mathematics, statistics and computer science, Johannes is developing Bioinformatic methods to improve the analysis of metagenomic data. He started his PhD studies in May 2014 after doing a master thesis project within the group.</p>
                                    <p>The first projects Johannes have been involved with conserns clustering of metagenomic contigs, reconstructing dominant baltic sea bacterioplankton and using microbial signatures for water quality assessment. Johannes is co-first author of CONCOCT and currently maintaining the code.</p>
								</section>

							</div>
						</div>
                        <div class="row">
                            <div class="6u">
                                <section>
                                    <a href="#" class="image feature"><img src="images/jurg.jpg" alt="" /></a>
                                    <header>
                                        <h3>Jürg Brendan Logue</h3>
                                    </header>
                                    <p>I am a PostDoctoral Research Fellow and am currently working on how bacterial communities of different aquatic origin differ in the processing of dissolved organic matter of terrestrial origin (in collaboration with Anders Andersson, KTH Stockholm, and Emma Kritzberg, Lund University).</p>
                                    <p>My general research interest lies in the ecology of micro-organisms within aquatic ecosystems. I am using microbial communities as model systems to address questions related to community ecology. I am particularly interested in understanding the mechanisms that underlie patterns of microbial species distribution, abundance, and interactions as well as the links between microbial diversity and ecosystem functioning.</p>
                                </section>
                            </div>
                            <div class="6u">
                                <section>
                                    <a href="#" class="image feature"><img src="images/yue.jpg" alt="" /></a>
                                    <header>
                                        <h3>Yue Hu</h3>
                                    </header>
                                    <p>Yue graduated from Wuhan University of Technology in China as Master of Environmental Science, and is now registered in KTH as a Ph.D student. She develops and uses metagenomics methods for environmental monitoring, including tracking contaminations in water systems and plankton monitoring in the Baltic Sea.</p> 
                                </section> 
                            </div> 
                        </div>
                    </section>

                    <section class="wrapper style1 container special", id="press">
                        <h2>Press Releases</h2>
                        <ul>
                            <li><a href="http://www.scilifelab.se/news/scilifelab-chosen-for-large-eu-project-to-improve-baltic-sea-health/">EU-project for microbial monitoring of the Baltic Sea</a></li>
                            <li><a href="http://www.liu.se/forskning/forskningsnyheter/1.489366?l=en">Decreased gut bacterial diversity in infants born by Caesarean section</a></li>
                            <li><a href="http://www.scilifelab.se/news/international-study-suggests-human-genes-influence-gut-microbial-composition/">A human gene associated with gut microbiota composition</a></li>
                            <li><a href="http://www.kth.se/en/aktuellt/nyheter/ostersjons-flora-av-bakterieplankton-kartlagd-1.178136">A 3D map of the Baltic Sea microbiome</a></li>
                        </ul>
                    </section>

                    <section class="wrapper style3 container special", id="projects">
                        <h2>Projects</h2>
                        <h3>Genomics and regulatory networks of Baltic Sea bacterioplankton communities</h3>
                        <p>The Baltic Sea suffers from environmental problems such as eutrophication and overfishing that are manifested in e.g. massive blooms of cyanobacteria in the summer and in oxygen depleted bottom waters.  In all marine environments, including the Baltic Sea, microbes are essential for the fluxes of nutrients and energy and are integral components of the food web. Despite their ecosystem importance, little is known about Baltic Sea microbial communities and how they are regulated. In collaboration with Jarone Pinhassi’s lab at the Linnaeus University we conduct a project where the surface water of an off-shore station east of Öland, the Linnaeus Microbial Observatory (LMO), is sampled twice weekly (started 2011, still ongoing). Samples from this station form the basis for two of our main projects concerning the Baltic Sea.
			</p>
                        <h3>Genomic reconstruction of dominant Baltic Sea bacterioplankton</h3>
                        <p>Although the sea is comprised of thousands of different microbial species, their abundance distributions are very skewed and a limited number of species make up most of the cells (20 species constitute >50% of the cells in surface water). Having the genomes of the more abundant populations sequenced will be important not only for the understanding of their specific roles in the ecosystem but also for understanding the ecosystem as a whole. We are performing shotgun metagenomics on a large number of the seasonal LMO samples. Overlapping short sequence reads are assembled into longer genome fragments. However, due to e.g. repeated sequences within and across the genomes, assembly results in fragmented genomes. A challenge is then to group fragments (contigs) stemming from the same genome. Our program CONCOCT that we recently developed with Christopher Quince and others (Alneberg et al 2014) allows us to, given that we have a data from multiple samples, group contigs into genomes with high accuracy. With this method and our extensive dataset we aim to present a catalogue of the most abundant microbial genomes of the Baltic Sea. As a proof of concept we used a similar approach to reconstruct the genome of one the most abundant bacteria in the Baltic Sea (Herlemann et al mBio 2013).
                        </p>
                        <h3>Reconstructing a regulatory network of Baltic Sea plankton assemblages</h3>
                        <p>By sequencing of PCR amplified taxonomic marker genes we can accurately measure the abundance of thousands of different bacterioplankton and microscopic eukaryotes in the seawater samples. For each species we will obtain a temporal profile over the years that the sampling has been on-going. In addition to the sequences we will have environmental data on temperature, nutrient levels etc. From this data the aim is to deduce the primary control mechanisms of the major plankton groups, such as bloom forming cyanobacteria. Preliminary data indicate that the different blooming events (spring bloom of eukaryotic phytoplankton and summer bloom of cyanobacteria) induce major state shifts in the bacterioplankton assemblages. Algorithms stemming from systems biology will be used to reconstruct a regulatory network of the ecosystem, consisting of environment-species and species-species interactions. We are also planning to use single cell analysis as a complementary approach to deduce species-species interactions.</p>
                        <h3>Exploring microbial signatures for water quality assessment</h3>
                        <p>Since microbial communities respond rapidly to changes in environmental conditions, both at the species composition and gene expression scale, they have great potential for being used as biomarkers of pollutants or other aspects of ecosystem health status. DNA sequencing technology develops extremely fast and within a few years it is likely that portable devices will be available for sequencing directly in the field. 
 
We are partners of the EU-funded project BLUEPRINT (Biological lenses using gene prints). The aim of this project is to use meta-omics for monitoring of Baltic Sea water health status. Based on metagenomic and/or transcriptomic data from hundreds of well characterized samples we will train algorithms to recognize different types of environmental conditions (such as early signs of oxygen depletion) based on the frequencies of taxonomic marker genes and/or genes coding for specific functions and/or metabolic pathways. We are responsible for the bioinformatics workpackage within this project. We are also participating in the EU-funded AFISmon project, where the aim is to develop an autonomous sampling device for collecting and storing water samples for metagenomics and metatranscriptomics analysis.
 
Together with Stockholm Water we are exploring the potential of metagenomic sand taxonomic marker gene sequencing for tracking contaminations of wastewater in the stonewater system of Stockholm. This may result from misconnections or leakages in the water systems, and can be highly problematic since wastewater may go un-cleaned into receiving water bodies. The goal is to find a faster and more precise alternative to the current method based on culturing the indicator species E. coli on selective media.</p>
                        <h3>Next generation plankton monitoring</h3>
                        <p>Unicellular plankton form the basis of marine food webs but they also include harmful members such as fish-killing species and producers of toxins that accumulate in shellfish. Quantitative assessment of unicellular plankton is therefore an important component of environmental monitoring programs. So far such monitoring has largely been based on microscopic counting of primarily phytoplankton and microzooplankton. This is a time-consuming process that requires highly skilled taxonomists, and pico- and nanoplankton, the most abundant planktonic fraction, often cannot be identified by this method. There is hence an urge for a faster, more robust and more precise way of quantifying plankton species in environmental samples. Together with SMHI we are conducting a project where we evaluate and optimize methodology for quantifying planktonic taxa in marine water samples by high-throughput DNA sequencing of taxonomic marker genes. Quantification based on microscope counting and sequencing will be conducted on samples collected at stations in the Baltic Sea, Kattegat and Skagerrak. The long-term goal is to incorporate the sequencing-based methodology in monitoring programs as a complement to traditional methods.</p>
                        <h3>Taxonomic binning of metagenomic contigs</h3>
                        <p>To be added</p>
                        <h3>Human Gut Microbiome</h3>
                        <p>Our bodies house more bacterial cells than own cells, most of which are located in the lower intestine. This microbiome helps the body extract energy from otherwise undigestible nutrient sources such as complex polysacharides. The microbiome also interacts with the immune system and associations between microbiota composition and various immune disorders have started to be unveiled. Together with Lars Engstrands group at the Karolinska Institute and Maria Jenmalms group at Linköping University we use taxonomic marker gene sequencing to investigate how the human gut microbiome develops in infants and how it is affected by factors such as mode of delivery. By following the children over several years we can relate microbiome diversity in infancy with health status later in life. We have shown that infants delivered by Cesarian section have a lower bacterial diversity in the gut early in life and that low diversity in infancy is associated with atopic eczema in infancy and asthma development at seven years of age.</p>
                        <h3>Metagenomics-driven enzyme discovery</h3>
                        <p>The great functional diversity of microbes (microbes can be found growing on virtually any substrate) opens up for discovery of biotechnology relevant enzymes within their genomes. Biomass from Swedish forests has a great potential for second-generation biofuel production as well as for bio-refinery products. In order to ferment the biomass into e.g. ethanol the lignocellulose first needs to be degraded to free the sugars. This can be achieved by using cocktails of carbohydrate active enzymes (CAZymes) but efficiencies of existing cocktails are rather low and not tailored towards Swedish forest biomass.  In collaboration with Henrik Aspeborg’s group at the Division of Industrial Biotechnology, KTH, and researchers at SLU, we have initiated a project aimed at finding and characterising novel CAZymes of the microbiome of the moose rumen. Branches and shoots of trees make up a substantial fraction of the moose diet, why the rumen of moose likely harbors a microbiome capable of degrading lignocellulose from Swedish trees. We find a large number of novel carbohydrate active enzymes in the bacterial genomes. These will be further characterised in silico and together with Aspeborg’s group selected novel genes will be expressed in the lab and their specificities and modes of action investigated.
                        </p>
                        <h3>Primer Design</h3>
                        <p>By sequencing of PCR amplified taxonomic marker genes we can accurately profile the microbial diversity within many samples in parallel. However, to minimize biases it is important to use primers that bind as non-selectively as possible. We have developed the program DegePrime (Hugerth et al - 1) that given a sequence alignmentfro each position finds the primer with a maximum degeneracy given by the user that matches as many sequences as possible. We used this to develop the primer pair 341F-805R that was show by another group to be the best primer pair for bacterial diversity studies out of 512 pairs tested. We also used DegePrime to greatly improve the coverage of a popular primer pair for assessing bacterial and archaeal diversity (515F-805R) and for developing a new primer pair for 18S rRNA gene surveys (574F-1123R) (Hugerth et al - 2).</p>
                    </section>
			</article>

		<!-- CTA -->
			<section id="cta">
			
				<header>
					<h2>Check out our projects on GitHub:</h2>
					<p>Where we share our bioinformatic software and lab protocols.</p>
				</header>
				<footer>
					<ul class="buttons">
						<li><a href="https://github.com/envgen" class="button special">EnvGen on Github</a></li>
					</ul>
				</footer>
			
			</section>


