---
layout: default
---
		<!-- Main -->
			<article id="metagenomics">
				<header class="special container">
					<span class="icon fa-bar-chart-o"></span>
                    <h2>Metagenomics Primer</h2>
                    <p>
                    Complex communities of microorganisms (microbiomes) play important roles in many ecosystems, such as oceans and soil, where they are the main drivers of most biogeochemical cycles. Moreover, the microbiome inhabiting the human gastrointestinal tract have lately been shown to play important roles for maintaining the health of its host. Since microbial communities are often comprised of thousands of species, most of which are very difficult to culture, studying them requires advanced technologies. Sequencing of the genomes of organisms, from viruses and bacteria to plants and animals, has fundamentally increased our understanding of different life forms. Recently, DNA sequencing has also proved to be extremely powerful for studying the genomes within microbiomes without prior cultivation - a method called metagenomics.  Metagenomic data reveals the species composition of a microbial community as well as its functional potential by decoding its functional genes.  Metagenomics is however a complex and technically demanding field with ample room for improvements in both laboratory and computational (bioinformatics) methodology.
                    </p>
				</header>
			    <section class="wrapper style4 container special", id="who">
					<a href="#"><img src="images/metagenomics_overview.jpg" alt="" /></a>
                    <header>
                        <h3>Metagenomics Outline</h3>
                    </header>
                </section>

                    <section class="wrapper style1 container special", id="press">
                        <p>A cheaper alternative to shotgun metagenomics is the sequencing of PCR amplified taxonomic marker genes, such as rRNA genes. This doesnâ€™t give any direct insights into the functional capabilities of the community but reveals its taxonomic composition. We typically sequence hundreds of samples at the same time and by using sample specific barcode sequences we can deduce from which sample the different sequence reads derive from. We typically get ~100,000 sequences per sample which allows us to quantify the different species with accuracy. Follow <a href="https://github.com/envgen">this link</a> for our lab and bioinformatics protocols for amplicon sequencing.</p>
                    </section>

                    <section class="wrapper style4 container special", id="projects">
                        <a href="#"><img src="images/amplicon_overview.jpg" alt="" /></a>
                        <header>
                            <h3>Amplicon sequencing of rRNA genes</h3>
                        </header>
                    </section>
			</article>

		<!-- CTA -->
			<section id="cta">
			
				<header>
					<h2>Check out our projects on GitHub:</h2>
					<p>Where we share our bioinformatic software and lab protocols.</p>
				</header>
				<footer>
					<ul class="buttons">
						<li><a href="https://github.com/envgen" class="button special">EnvGen on Github</a></li>
					</ul>
				</footer>
			
			</section>


