---
layout: default
---
			<section id="banner">
				
				<!--
					".inner" is set up as an inline-block so it automatically expands
					in both directions to fit whatever's inside it. This means it won't
					automatically wrap lines, so be sure to use line breaks where
					appropriate (<br />).
				-->
				<div class="inner">
					
					<header>
                        <img src="images/logo.png"/>
					</header>
                    <p>
                    This page contains descriptions for all <br/>
                    the projects we're involved in.
                    </p>
					<footer>
						<ul class="buttons">
							<li><a href="/" class="button fit scrolly">Go Back to Front Page</a></li>
							<li><a href="#main" class="button fit scrolly">Tell Me More</a></li> 
						</ul>
					</footer>
				
				</div>
				
			</section>

<!-- Main -->
			<article id="main">
            <section class="wrapper style1 container special">
                <h2>Projects</h2>
		<h3>Evolution and biogeography of the brackish microbiome</h3>
                <p>In all aquatic environments, microorganisms are the engines that drive the fluxes of energy and nutrients and form the base of the food web. Due to climate change, temperature and salinity will change in many water bodies. To predict how ecosystems will change, knowledge on adaptation and redistribution of microbes in response to altered environment conditions is central. The young (8000yr.) Baltic Sea with its strong environmental gradients is a perfect system for studying microbial colonisation and adaptation. A large body of research has investigated how macro-organisms have adapted to the brackish conditions of this sea. However, little is known about how microorganisms colonised and adapted to this ecosystem. This project will shed light on the evolutionary history and biogeography of brackish-water prokaryotes by leveraging on a comprehensive catalogue of metagenome-assembled genomes (MAGs) from globally distributed freshwater, brackish and marine environments. Phylogenomic and population genomic approaches will be combined with analysis of ancient DNA in sediments and DNA from air filters. The project will shed light on how brackish prokaryotes evolved from their freshwater and marine ancestors, how they differentiate within and disperse between ecosystems, and how the Baltic Sea microbiome has developed over geological time. Using the brackish microbiome as a model, it will provide insights on how microbial populations evolve, differentiate and disperse in nature. (Description taken from the (<a href="https://www.vr.se/english/swecris.html#/project/2021-05563_VR">Swedish Research Council's website</a>)</p>
                <h3>Genomics and regulatory networks of Baltic Sea bacterioplankton communities</h3>
                <p>The Baltic Sea suffers from environmental problems such as eutrophication and overfishing that are manifested in e.g. massive blooms of cyanobacteria in the summer and in oxygen depleted bottom waters.  In all marine environments, including the Baltic Sea, microbes are essential for the fluxes of nutrients and energy and are integral components of the food web. Despite their ecosystem importance, little is known about Baltic Sea microbial communities and how they are regulated. In collaboration with Jarone Pinhassi’s lab at the Linnaeus University we conduct a project where the surface water of an off-shore station east of Öland, the Linnaeus Microbial Observatory (LMO), is sampled twice weekly (started 2011, still ongoing). Samples from this station form the basis for two of our main projects concerning the Baltic Sea.</p>
                 <h3>Genomic reconstruction of dominant Baltic Sea bacterioplankton</h3>
                 <p>Although the sea is comprised of thousands of different microbial species, their abundance distributions are very skewed and a limited number of species make up most of the cells (20 species constitute >50% of the cells in surface water). Having the genomes of the more abundant populations sequenced will be important not only for the understanding of their specific roles in the ecosystem but also for understanding the ecosystem as a whole. We are performing shotgun metagenomics on a large number of the seasonal LMO samples. Overlapping short sequence reads are assembled into longer genome fragments. However, due to e.g. repeated sequences within and across the genomes, assembly results in fragmented genomes. A challenge is then to group fragments (contigs) stemming from the same genome. Our program CONCOCT that we recently developed with Christopher Quince and others (<a href="http://www.ncbi.nlm.nih.gov/pubmed/25218180">Alneberg & Bjarnason et al 2014</a>) allows us to, given that we have a data from multiple samples, group contigs into genomes with high accuracy. With this method and our extensive dataset we aim to present a catalogue of the most abundant microbial genomes of the Baltic Sea. As a proof of concept we used a similar approach to reconstruct the genome of one the most abundant bacteria in the Baltic Sea (<a href="http://www.ncbi.nlm.nih.gov/pubmed/23716574">Herlemann et al mBio 2013</a>).
                  </p>
                  <p>
                  <a href="#"><img src="images/baltic1.jpg" alt="" /></a>
                  </p>
                  <br/>
                  <h3>Reconstructing a regulatory network of Baltic Sea plankton assemblages</h3>
                  <p>By sequencing of PCR amplified taxonomic marker genes we can accurately measure the abundance of thousands of different bacterioplankton and microscopic eukaryotes in the seawater samples. For each species we will obtain a temporal profile over the years that the sampling has been on-going. In addition to the sequences we will have environmental data on temperature, nutrient levels etc. From this data the aim is to deduce the primary control mechanisms of the major plankton groups, such as bloom forming cyanobacteria. Preliminary data indicate that the different blooming events (spring bloom of eukaryotic phytoplankton and summer bloom of cyanobacteria) induce major state shifts in the bacterioplankton assemblages. Algorithms stemming from systems biology will be used to reconstruct a regulatory network of the ecosystem, consisting of environment-species and species-species interactions. We are also planning to use single cell analysis as a complementary approach to deduce species-species interactions.</p>
                  <h3>Exploring microbial signatures for water quality assessment</h3>
                  <p>Since microbial communities respond rapidly to changes in environmental conditions, both at the species composition and gene expression scale, they have great potential for being used as biomarkers of pollutants or other aspects of ecosystem health status. DNA sequencing technology develops extremely fast and within a few years it is likely that portable devices will be available for sequencing directly in the field. 
 
We are partners of the EU-funded project BLUEPRINT (Biological lenses using gene prints). The aim of this project is to use meta-omics for monitoring of Baltic Sea water health status. Based on metagenomic and/or transcriptomic data from hundreds of well characterized samples we will train algorithms to recognize different types of environmental conditions (such as early signs of oxygen depletion) based on the frequencies of taxonomic marker genes and/or genes coding for specific functions and/or metabolic pathways. We are responsible for the bioinformatics workpackage within this project. We are also participating in the EU-funded AFISmon project, where the aim is to develop an autonomous sampling device for collecting and storing water samples for metagenomics and metatranscriptomics analysis.
 
Together with Stockholm Water we are exploring the potential of metagenomic sand taxonomic marker gene sequencing for tracking contaminations of wastewater in the stonewater system of Stockholm. This may result from misconnections or leakages in the water systems, and can be highly problematic since wastewater may go un-cleaned into receiving water bodies. The goal is to find a faster and more precise alternative to the current method based on culturing the indicator species E. coli on selective media.</p>
                   <h3>Next generation plankton monitoring</h3>
                   <p>Unicellular plankton form the basis of marine food webs but they also include harmful members such as fish-killing species and producers of toxins that accumulate in shellfish. Quantitative assessment of unicellular plankton is therefore an important component of environmental monitoring programs. So far such monitoring has largely been based on microscopic counting of primarily phytoplankton and microzooplankton. This is a time-consuming process that requires highly skilled taxonomists, and pico- and nanoplankton, the most abundant planktonic fraction, often cannot be identified by this method. There is hence an urge for a faster, more robust and more precise way of quantifying plankton species in environmental samples. Together with SMHI we are conducting a project where we evaluate and optimize methodology for quantifying planktonic taxa in marine water samples by high-throughput DNA sequencing of taxonomic marker genes. Quantification based on microscope counting and sequencing will be conducted on samples collected at stations in the Baltic Sea, Kattegat and Skagerrak. The long-term goal is to incorporate the sequencing-based methodology in monitoring programs as a complement to traditional methods.</p>
                   <h3>Taxonomic binning of metagenomic contigs</h3>
                   <p>Metagenomic sequencing can successfully produce genome fragments (contigs) from complex microbial communities through assembly, but since the assembly does not reconstruct entire genomes, binning of contigs is necessary. Traditionally, unsupervised metagenomic binning programs have used sequence composition, exploiting k-mer biases between species, and/or coverage differences within a sample. Together with Christopher Quince and others we developed the software CONCOCT, Clustering cONtigs on COverage and ComposiTion, (<a href="http://www.ncbi.nlm.nih.gov/pubmed/25218180">Alneberg & Bjarnason et al 2014</a>). CONCOCT combines sequence compisition and coverage, two sources of information commonly used in binning but also extends the way coverage is commonly used. By using patterns of coverage distributions over multiple samples instead of a single one a stronger signal for clustering is achieved. Using Bayesian statistics to estimate the parameters of a Gaussian Mixture model, CONCOCT can automatically cluster contigs into genomes without the need to fix the number of clusters in beforehand. </p>
                   <h3>Human Gut Microbiome</h3>
                   <p>Our bodies house more bacterial cells than own cells, most of which are located in the lower intestine. This microbiome helps the body extract energy from otherwise undigestible nutrient sources such as complex polysacharides. The microbiome also interacts with the immune system and associations between microbiota composition and various immune disorders have started to be unveiled. Together with Lars Engstrands group at the Karolinska Institute and Maria Jenmalms group at Linköping University we use taxonomic marker gene sequencing to investigate how the human gut microbiome develops in infants and how it is affected by factors such as mode of delivery. By following the children over several years we can relate microbiome diversity in infancy with health status later in life. We have shown that infants delivered by Cesarian section have a lower bacterial diversity in the gut early in life (<a href="http://www.ncbi.nlm.nih.gov/pubmed/23926244">Jakobsson et al 2014</a>) and that low diversity in infancy is associated with atopic eczema in infancy (<a href="http://www.ncbi.nlm.nih.gov/pubmed/22153774">Abrahamson et al 2012</a>) and asthma development at seven years of age (<a href="http://www.ncbi.nlm.nih.gov/pubmed/24330256">Abrahamson et al 2014</a>).</p>

                    <h3>Metagenomics-driven enzyme discovery</h3>
                    <p>The great functional diversity of microbes (microbes can be found growing on virtually any substrate) opens up for discovery of biotechnology relevant enzymes within their genomes. Biomass from Swedish forests has a great potential for second-generation biofuel production as well as for bio-refinery products. In order to ferment the biomass into e.g. ethanol the lignocellulose first needs to be degraded to free the sugars. This can be achieved by using cocktails of carbohydrate active enzymes (CAZymes) but efficiencies of existing cocktails are rather low and not tailored towards Swedish forest biomass.  In collaboration with Henrik Aspeborg’s group at the Division of Industrial Biotechnology, KTH, and researchers at SLU, we have initiated a project aimed at finding and characterising novel CAZymes of the microbiome of the moose rumen. Branches and shoots of trees make up a substantial fraction of the moose diet, why the rumen of moose likely harbors a microbiome capable of degrading lignocellulose from Swedish trees. We find a large number of novel carbohydrate active enzymes in the bacterial genomes. These will be further characterised in silico and together with Aspeborg’s group selected novel genes will be expressed in the lab and their specificities and modes of action investigated.
                    </p>
                    <p>
                    <a href="#"><img src="images/moose1.jpg" alt="" /></a>
                    </p>
                    
                    <h3>Primer Design</h3>
                    <p>By sequencing of PCR amplified taxonomic marker genes we can accurately profile the microbial diversity within many samples in parallel. However, to minimize biases it is important to use primers that bind as non-selectively as possible. We have developed the program DegePrime (<a href="http://www.ncbi.nlm.nih.gov/pubmed/24928874">Hugerth et al - 1</a>) that given a sequence alignmentfro each position finds the primer with a maximum degeneracy given by the user that matches as many sequences as possible. We used this to develop the primer pair 341F-805R that was show by another group to be the best primer pair for bacterial diversity studies out of 512 pairs tested. We also used DegePrime to greatly improve the coverage of a popular primer pair for assessing bacterial and archaeal diversity (515F-805R) and for developing a new primer pair for 18S rRNA gene surveys (574F-1123R) (<a href="http://www.ncbi.nlm.nih.gov/pubmed/24755918">Hugerth et al - 2</a>).</p>

					<h3>Artificial intelligence-assisted plankton Monitoring with Imaging flow cytometry</h3>
                <p>Single-celled eukaryotic plankton (protists) form the productive base of marine ecosystems and are key drivers of global biogeochemical cycles of carbon and nutrients. Consequently, plankton have fundamental impacts on both fish stocks and the climate and understanding the factors that control the abundance and distribution of different
plankton species is of great concern. Monitoring of eukaryotic plankton has traditionally been conducted by
manual microscopic detection. Recently, alternative approaches have emerged such as high-throughput imaging
and DNA metabarcoding. While promising, these methods have their challenges, not least in how to translate
between these disparate datasets. In this project we will utilize state-of-the-art image analysis and deep learning
approaches to maximise the information gained from these types of data and to translate between them. We will
leverage on existing imaging data from the new Imaging FlowCytobot (IFCB) instrument mounted on the research
vessel Svea as well as generate new parallel IFCB and DNA metabarcoding datasets for 500 water samples
spanning the Baltic Sea, Kattegat and Skagerrak. The methodology developed in the project will advance plankton
research and ecology in general and plankton monitoring in Sweden in particular. It will bridge the gap between
imaging and DNA-based diversity data and increase the information output from both approaches.</p>

				<h3 id="visible-project">Virus-host interactions in the Baltic Sea (VISIBLE)</h3>
				<p>VISIBLE – Virus-Host Interactions in the Baltic Sea – is an EU-funded Marie Skłodowska-Curie Actions project that explores interactions between bacteria and phages in the marine environment using meta-omic approaches. Using hundreds of short- and long-read seawater metagenomes and viromes from the Linnaeus Microbial Observatory (LMO), we link phages to their bacterial hosts over a nine-year time series and investigate their coupled population dynamics over time.
VISIBLE is conducted in collaboration with Jarone Pinhassi and Karin Holmfeldt from Linnaeus University.
</p>
				<h3 id="sealab-project">SeaLab at SciLifeLab</h3>
				<p><a href="https://www.su.se/english/research/research-catalogue/research-projects/9/sealab">SeaLab</a> is a collaborative research initiative between the Baltic Sea Centre at Stockholm University and the Environmental Genomics group at SciLifeLab. By integrating expertise in biogeochemistry, ecology, and meta-omics, the goal is to uncover the fundamental microbial processes that shape coastal ecosystems. 
SeaLab projects focus on microbial metabolic processes that drive the cycling of climate-active compounds in coastal environments, with particular emphasis on greenhouse gases (GHGs) and volatile organic compounds (VOCs) in both sediments and the water column. We also collect regular (~monthly) seawater samples for meta-omic analyses from the Askö monitoring station B1 together with Askö laboratory. 
</p>

                    </section>
			</article>

		<!-- CTA -->
			<section id="cta">
			
				<header>
					<h2>Check out our projects on GitHub:</h2>
					<p>Where we share our bioinformatic software and lab protocols.</p>
				</header>
				<footer>
					<ul class="buttons">
						<li><a href="https://github.com/envgen" class="button special">EnvGen on Github</a></li>
					</ul>
				</footer>
			
			</section>


